packages/app/src/hooks/useChartConfig.tsx

```diff
@@ -13,6 +15,49 @@ import { IS_MTVIEWS_ENABLED } from '@/config';
 import { buildMTViewSelectQuery } from '@/hdxMTViews';
 import { getMetadata } from '@/metadata';
 
+export const isMetric = (config: ChartConfigWithOptDateRange) =>
+  config.metricTables != null;
+
+// TODO: apply this to all chart configs
+export const setChartSelectsAlias = (config: ChartConfigWithOptDateRange) => {
+  if (Array.isArray(config.select)) {
+    if (isMetric(config)) {
+      return {
+        ...config,
+        select: config.select.map(s => ({
+          ...s,
+          alias: `${s.aggFn}(${s.metricName})`,
+        })),
+      };
+    }
+    return {
+      ...config,
+      select: config.select.map(s => ({
+        ...s,
+        alias: `${s.aggFn}(${s.valueExpression})`,
+      })),
+    };
+  }
+  return config;
+};
+
+export const splitChartConfigs = (config: ChartConfigWithOptDateRange) => {
+  // only split metric queries for now
```

Comment:
what other queries would we want to do this on?

Reply:
I put this here in case we need to split the queries somewhere else. currently only for `useQueriedChartConfig`